app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.players.player import Player
+from typing import List
+
+
+def calculate_team_total_rating(team: List[Player]) -> int:
+    return sum(rating.get_rating() for rating in team)
+
+
+def elves_concert(elves: list) -> None:
+    for elf in elves:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarves: list) -> None:
```

Comment:
Consider using `List` from the `typing` module for type annotations instead of the built-in `list`. This will ensure consistency with the `calculate_team_total_rating` function and adhere to the checklist's recommendation for using annotations.